<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hotspots</class>
 <widget class="QWidget" name="Hotspots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>700</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>1011</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hotspots</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>1011</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>650</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>De</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelFechaIni">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ini</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>a</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelFechaFin">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>fin</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1251</width>
     <height>603</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="verticalFrame1">
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>109</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arimo</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Posiciones</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textInfoL">
         <property name="maximumSize">
          <size>
           <width>111</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arimo</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Graph" name="openGLWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>601</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="verticalFrame2">
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>109</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arimo</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Posiciones</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textInfoR">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>109</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arimo</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>650</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label6">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTotalN">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>n</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Graph</class>
   <extends>QOpenGLWidget</extends>
   <header>graph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
